<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLaunch</class>
 <widget class="QDialog" name="DialogLaunch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>agtv - launcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="agtv.qrc">
    <normaloff>:/images/agtv_32.png</normaloff>:/images/agtv_32.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxPosSelect"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelPos">
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelStream">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="agtv.qrc"/>
 </resources>
 <connections/>
</ui>
